aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -131,12 +133,12 @@ const StepOne = () => {
     WizardContext,
   );
   const size = useContext(ResponsiveContext);
-  console.log(error.isValid);
+
   return (
     <>
       <Box margin={{ bottom: 'medium' }}>
```

Comment:
Can we eliminate either the Fragment or Box? I don't think we need both.

Reply:
Yes, can eliminate the Fragment, thanks.